clang/runtime/dpct-rt/include/dpct/memory.hpp

```diff
@@ -1676,6 +1685,40 @@ class pointer_attributes {
 #endif
   }
 
+  static void get(unsigned int numAttributes, type *attributes,
```

Comment:
Please add comment for this publich interface, this function is not so straightforward.